wtr-utils.js

```diff
@@ -293,6 +293,7 @@ const createVisualTestsConfig = (theme, browserVersion) => {
         browserName: 'chrome',
         platformName: 'Windows 10',
         browserVersion,
+        'wdio:enforceWebDriverClassic': true,
```

Comment:
I had to enable this option since `24.6` branch uses Chrome versions 110 (Lumo) and 88 (Material) and apparently these don't support BiDi protocol used by default in Webdriver 9.